fe1-web/src/resources/strings.ts

```diff
@@ -305,7 +311,7 @@ namespace STRINGS {
     'The Roll Call is currently open and you as the organizer should start adding attendees by scanning their PoP tokens.';
   export const roll_call_open_attendee =
     'The Roll Call is currently open and you as an attendee should let the organizer scan your PoP token encoded in the QR Code below.';
-  export const roll_call_qrcode_text = 'Scan to add attendee';
+  export const roll_call_qrcode_text = 'Scan to\nadd\nattendee';
```

Comment:
Are the newlines necessary? Did the automatic text wrapping break?

Reply:
For me it didn't work here in qr code overlay

Reply:
Oh okay this is not good 😅 But I guess this is a good fix for now